src/sas/qtgui/Calculators/GenericScatteringCalculator.py

```diff
@@ -633,7 +634,7 @@ def update_cbOptionsCalc_visibility(self):
             self.checkboxLogSpace.setEnabled(not self.is_mag)
 
 
-    def change_is_avg(self):
+    def change_computation_type(self):
```

Comment:
Nitpick: It's often a bad idea to completely replace a method, in case there are direct calls to it that may have been missed.

```suggestion
    def change_is_avg(self):
        self.change_computation_type()

    def change_computation_type(self):
```